### PR TITLE
Adding submission review values to swagger

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

Adding review values to openapi and up date mappings

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

## Does the change impact or break the Docker build?

- [ ] Yes
- [X] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [X] No

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/efiling-api/jag-efiling-api.yaml [hunk 1]
```diff
@@ -1820,6 +1820,18 @@ components:
         mimeType:
           type: string
           description: File mime type
+        status:
```
Comment (alexjoybc) [inline src/backend/efiling-api/jag-efiling-api.yaml:1823]: status as an object

**Review by alexjoybc — CHANGES_REQUESTED** [review]
change for the status object ;-)

File: src/backend/efiling-api/jag-efiling-api.yaml (hunk 1 above)
Comment (TayGov) [reply src/backend/efiling-api/jag-efiling-api.yaml:1823]: agreed

File: src/backend/efiling-api/jag-efiling-api.yaml [hunk 2]
```diff
@@ -2311,3 +2329,38 @@ components:
...
+      description: document status
+      type: object
+      properties:
+        description:
+          type: string
+          description: description of status
+        code:
+          type: string
+          description: status identifier
+        date:
```
Comment (alexjoybc) [inline src/backend/efiling-api/jag-efiling-api.yaml:2364]: Can we change this to `somethingDate` autogen might not like `date`

**Review by alexjoybc — CHANGES_REQUESTED** [review]
small detail with the date

**Review by alexjoybc — APPROVED** [review]

**Pull request merged**